### PR TITLE
EOS-14369 Build #303 : S3 sanity failed after successful build deploy…

### PR DESCRIPTION
**Pull request by abhilekh268**
…ment

Signed-off-by: abhilekh <abhilekh.mustapure@seagate.com>

### CODE REVIEW

File: s3-sanity-test.sh [hunk 1]
```diff
@@ -58,11 +58,13 @@ then
...
 
-USAGE="USAGE: bash $(basename "$0") [--help]
+USAGE="USAGE: bash $(basename "$0")
 Run S3 sanity test.
 where:
-    --help      display this help and exit
-    --clean     clean s3 resources if present
+    -h     display this help and exit
+    -c     clean s3 resources if present
+    -e     specify the server endpoint.it should be localhost(127.0.0.1)
```
Comment (bkirunge7) [inline s3-sanity-test.sh:66]: specify the s3 server endpoint, pass localhost if running on s3 server node

File: s3-sanity-test.sh [hunk 2]
```diff
@@ -93,24 +95,33 @@ cleanup() {
...
+    esac
+done
+shift $((OPTIND-1))
 
 trap "cleanup" ERR
 
 echo -e "\n\n*** S3 Sanity ***"
 echo -e "\n\n**** Create Account *******"
 
+if [ ! -z $end_point ];then
```
Comment (bkirunge7) [inline s3-sanity-test.sh:120]: Print "Using S3 Endpoint <endpoint value>"

File: s3-sanity-test.sh [hunk 3]
```diff
@@ -93,24 +95,33 @@ cleanup() {
...
+  sed -i "s/IAM:.*/IAM: $end_point:9080/g" /root/.sgs3iamcli/config.yaml
+  sed -i "s/IAM_HTTPS:.*/IAM_HTTPS: $end_point:9043/g" /root/.sgs3iamcli/config.yaml
```
Comment (AjinkyaDhumal) [inline s3-sanity-test.sh:122]: I think the port no. should be 9443 ?

File: s3-sanity-test.sh (hunk 1 above)
Comment (abhilekh268) [reply s3-sanity-test.sh]: done

File: s3-sanity-test.sh (hunk 2 above)
Comment (abhilekh268) [reply s3-sanity-test.sh:119]: done

File: s3-sanity-test.sh (hunk 3 above)
Comment (abhilekh268) [reply s3-sanity-test.sh]: done

**Review by sachitanands — APPROVED** [review]
please test these changes once on h/w ..

File: s3-sanity-test.sh [hunk 4]
```diff
@@ -161,5 +173,5 @@ set +e
 
 # delete test files file
 rm -f $test_file_input $test_output_file
-
+sed -i "s/VERIFY_SSL_CERT:.*/VERIFY_SSL_CERT: True/g" /root/.sgs3iamcli/config.yaml
```
Comment (bkirunge7) [inline s3-sanity-test.sh:176]: add this in cleanup as well

**Review by bkirunge7 — APPROVED** [review]

**Pull request merged**